tools/component-builder-simple/css/processors.js

```diff
@@ -30,6 +30,7 @@ function getProcessors({ noFlatVariables = false, noSelectors = false, keepComme
 		require("postcss-dropunusedvars")({ fix: false }),
 		require("postcss-dropdupedvars"),
 		require("postcss-discard-empty"),
+    require("postcss-hover-media-feature"),
```

Comment:
Super minor knit and you don't have to fix this before we merge but the new imports are not aligned with the others in the list.